### ISSUE TITLE
Can you provide the mat file of the detection bounding boxes of the itop side and top training set

### ISSUE THREAD

**Issue by liukuixiang** [zhangboshen/A2J#49]
hello， Can you provide the mat file of the detection bounding boxes of the itop side and top training set？ look forward to your reply，thanks！

**Comment by Tekicho** [comment]
While attempting to reproduce the results of A2J, by retraining on NYU dataset, I am getting regression loss and total loss as "inf". What could be wrong. I am using     pytorch1.5.1 cuda10.1, cudnn7.6.1 on windows10.

**Comment by zhangboshen** [comment]
Hi, @liukuixiang ，in our implementation, the bounding box for itop dataset during training is simply extracted via the GT keypoints annotations.
